### PR TITLE
enable contact notifications for all

### PR DESCRIPTION
**Pull request by joshm1**
and decrement social refresh interval  from 11 -> 9 days

@stkem

### CODE REVIEW

**Comment by stkem** [comment]
lgtm

**Comment by joshm1** [comment]
going to add tracking before I merge

File: kifi-backend/modules/shoebox/app/com/keepit/model/SocialUserInfoRepo.scala [hunk 1]
```diff
@@ -58,7 +58,7 @@ class SocialUserInfoRepoImpl @Inject() (
 
   private val UNPROCESSED_STATES = SocialUserInfoStates.CREATED :: SocialUserInfoStates.FETCHED_USING_FRIEND :: Nil
   private val REFRESHING_STATES = SocialUserInfoStates.FETCHED_USING_SELF :: SocialUserInfoStates.FETCH_FAIL :: Nil
-  private val REFRESH_FREQUENCY = 11 // days TODO (josh) - decrement this by 1 or 2 every day until it = 2
+  private val REFRESH_FREQUENCY = 9 // days TODO (josh) - decrement this by 1 or 2 every day until it = 2
```
Comment (eishay) [inline kifi-backend/modules/shoebox/app/com/keepit/model/SocialUserInfoRepo.scala:61]: what does it mean? why not having it = 2?

Comment (joshm1) [inline kifi-backend/modules/shoebox/app/com/keepit/model/SocialUserInfoRepo.scala:61]: if we change it immediately from 15 (which it started at) to 2, the next deploy to kick off social network updates at once for most users.  This was just a quick/lazy solution to get the # down w/o putting everybody on the same refresh cycle.  Another PR today however adds the ability to randomly spread all refreshes over 2 days, so after I run that, I will just update this to 2 and be done.

Comment (eishay) [inline kifi-backend/modules/shoebox/app/com/keepit/model/SocialUserInfoRepo.scala:61]: cool, thanks

**Pull request merged**